apps/dotcms-ui/src/app/view/components/dot-content-compare/components/dot-content-compare-table/dot-content-compare-table.component.html

```diff
@@ -1,73 +1,130 @@
-<p-table [value]='data.fields' *ngIf='data'  styleClass="p-datatable-gridlines">
-    <ng-template pTemplate='header'>
+<p-table [value]="data.fields" *ngIf="data" styleClass="p-datatable-gridlines">
+    <ng-template pTemplate="header">
         <tr>
-            <th style="width:12%"> </th>
+            <th style="width: 12%"></th>
             <th>
                 <span data-testId="table-tittle" class="dot-content-compare-table__title">
-                    {{data.working?.title}}
+                    {{ data.working?.title }}
                 </span>
             </th>
             <th>
-               <div class="dot-content-compare-table__controls">
-                   <p-dropdown [options]="data.versions" (onChange)='changeVersion.emit($event.value)' appendTo="body" >
-                       <ng-template let-selected pTemplate="selectedItem">
-                           {{selected.modDate}}
-                       </ng-template>
-                       <ng-template let-item pTemplate="item">
-                           {{item.modDate}}
-                       </ng-template>
-                   </p-dropdown>
-                   <p-selectButton
-                       [options]="displayOptions"
-                       [(ngModel)]="showDiff"
-                       class="p-button-tabbed"
-                       (onChange)="changeDiff.emit($event.value)" >
-                   </p-selectButton>
-                   <button data-testId="table-bring-back" pButton class="p-button-sm p-button" [label]="'Bring-Back' | dm" (click)='bringBack.emit(data.compare.inode)'></button>
-               </div>
+                <div class="dot-content-compare-table__controls">
+                    <p-dropdown
+                        [options]="data.versions"
+                        data-testid="versions-dropdown"
+                        (onChange)="changeVersion.emit($event.value)"
+                        appendTo="body"
+                    >
+                        <ng-template let-selected pTemplate="selectedItem">
+                            {{ transformVersionItemLabel(selected) }}
```

Comment:
https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496